--- ./android/app/build.gradle	2023-12-08 11:17:36.130601770 +0100
+++ ./android/app/build_split_abi.gradle	2023-12-08 11:21:11.144304948 +0100
@@ -79,9 +79,9 @@
     buildTypes {
         release {
             signingConfig signingConfigs.release
-            ndk {
-                abiFilters "armeabi-v7a", "arm64-v8a", "x86_64"
-            }
+//            ndk {
+//                abiFilters "armeabi-v7a", "arm64-v8a", "x86_64"
+//            }
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
