--- ./android/app/build.gradle.kts	2025-06-12 17:50:25.720695603 +0200
+++ ./android/app/build_split_abi.gradle.kts	2025-06-12 17:59:55.119389553 +0200
@@ -73,9 +73,9 @@
     buildTypes {
         release {
             signingConfig = signingConfigs.getByName("release")
-            ndk {
-                abiFilters.addAll(arrayOf("armeabi-v7a", "arm64-v8a", "x86_64"))
-            }
+//            ndk {
+//                abiFilters.addAll(arrayOf("armeabi-v7a", "arm64-v8a", "x86_64"))
+//            }
             isMinifyEnabled = true
             proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
         }
